src/components/common/elements/slider/style.css

```diff
@@ -0,0 +1,8 @@
+.rc-slider-track {
```

Comment:
Let's avoid adding a separate CSS for this. You can merge this in `src/components/common/blocks/gas-price/style.js` and use the container of the component as a selector. So it should look something like this:

```js
export const SliderCard = styled(Card)`
  /* insert styles for SliderCard here */

  .rc-slider-track {
    /* insert styles to override default library styles here */
  }
`;
```